deploy/ray-operator/Makefile

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
If possible, it would be great to integrate this into the bazel build system that we use for the rest of ray. Shouldn't be too much effort. See https://github.com/bazelbuild/rules_go to get started.

Reply:
Yes, I am working on how to integrate bazel.